test_helpers.py

```diff
@@ -0,0 +1,206 @@
+''' This file contains the unit tests for the helper functions in the helpers.py file. '''
```

Comment:
I've never done "real" and clean unit tests in Python. Great to have a start, now!

I haven't read this yet. Is it intended to completely replace tests.py (maybe later)? Would be great!

Reply:
The tests.py is right know more an integration test which is pretty good. Wen can think about this but for know it should be fine. I try to add some more unit tests in future.

Reply:
Yes. I agee.
(I started the tests to have at least more than nothing)